crates/next-api/src/next_server_nft.rs

```diff
@@ -0,0 +1,340 @@
+        })
+        .chain(resolve_entry("next/dist/server/next-server").await?)
+        .map(|m| Vc::upcast::<Box<dyn OutputAsset>>(TracedAsset::new(m)).to_resolved())
+        .try_join()
+        .await?;
+
+    let minimal_server_entries = shared_entries
+        .iter()
+        .chain(cache_handler_entries.iter())
+        .copied()
+        .chain(resolve_entry("next/dist/compiled/next-server/server.runtime.prod").await?)
+        .map(|m| Vc::upcast::<Box<dyn OutputAsset>>(TracedAsset::new(m)).to_resolved())
+        .try_join()
+        .await?;
+
+    let output_file_tracing_excludes = project.next_config().output_file_tracing_excludes().await?;
+    let mut additional_ignores = BTreeSet::new();
+    if let Some(output_file_tracing_excludes) = output_file_tracing_excludes
+        .as_ref()
+        .and_then(Value::as_object)
+    {
+        for (glob_pattern, exclude_patterns) in output_file_tracing_excludes {
+            // Check if the route matches the glob pattern
+            let glob = Glob::new(RcStr::from(glob_pattern.clone()), Default::default()).await?;
+            if glob.matches("next-server")
+                && let Some(patterns) = exclude_patterns.as_array()
+            {
+                for pattern in patterns {
+                    if let Some(pattern_str) = pattern.as_str() {
+                        additional_ignores.insert(pattern_str);
+                    }
+                }
+            }
+        }
+    }
+
+    let server_ignores_glob = [
+        "**/node_modules/react{,-dom,-server-dom-turbopack}/**/*.development.js",
+        "**/*.d.ts",
+        "**/*.map",
+        "**/next/dist/pages/**/*",
+        "**/next/dist/compiled/next-server/**/*.dev.js",
+        "**/next/dist/compiled/webpack/*",
+        "**/node_modules/webpack5/**/*",
+        "**/next/dist/server/lib/route-resolver*",
+        "**/next/dist/compiled/semver/semver/**/*.js",
+        "**/next/dist/compiled/jest-worker/**/*",
+        // Turbopack doesn't support AMP
+        "**/next/dist/compiled/@ampproject/toolbox-optimizer/**/*",
+        // -- The following were added for Turbopack specifically --
+        // client/components/use-action-queue.ts has a process.env.NODE_ENV guard, but we can't set that due to React: https://github.com/vercel/next.js/pull/75254
+        "**/next/dist/next-devtools/userspace/use-app-dev-rendering-indicator.js",
+        // client/components/app-router.js has a process.env.NODE_ENV guard, but we can't set that.
+        "**/next/dist/client/dev/hot-reloader/app/hot-reloader-app.js",
+        // server/lib/router-server.js doesn't guard this require:
+        "**/next/dist/server/lib/router-utils/setup-dev-bundler.js",
+        // server/next.js doesn't guard this require
+        "**/next/dist/server/dev/next-dev-server.js",
+        // next/dist/compiled/babel* pulls in this, even we never actually transpile with  Babel
+        "**/next/dist/compiled/browserslist/**",
```

Comment:
We never *downlevel* with babel, but we do compile with Babel (for react-compiler, but also I'm working on fixing `.babelrc` support).

But yeah, it's a compile-time dependency at best, not a runtime dependency.